public/content/developers/docs/data-and-analytics/index.md

```diff
@@ -1,55 +1,122 @@
+If you are new to web3 and this looks confusing to you, think of these web2 parallels:
 
-## The Graph {#the-graph}
+- indexing: amplitude, stripe, general logging services
+- explore: grafana, datadog
+- query: metabase, hex
+- define and store: dbt, snowflake, databricks
+- …and back onchain: this is just reverse ETL
 
-The [Graph Network](https://thegraph.com/) is a decentralized indexing protocol for organizing blockchain data. Instead of building and managing off-chain and centralized data stores to aggregate on-chain data, with The Graph, developers can build serverless applications that run entirely on public infrastructure.
+### Index: Read raw data from the blockchain
 
-Using [GraphQL](https://graphql.org/), developers can query any of the curated open APIs, known as sub-graphs, to acquire the necessary information they need to drive their dapp. By querying these indexed sub-graphs, Reports and dapps not only get performance and scalability benefits but also the built in accuracy provided by network consensus. As new improvements and/or sub-graphs are added to the network, your projects can rapidly iterate to take advantage of these enhancements.
+Blockchains run with nodes. Nodes run “client” code - which are repos that have implemented the EVM in some fashion. These clients have a set of implemented RPCs (API endpoints), some of which are standard and some of which are custom to support better/faster data querying. Data from RPC endpoints are direct reflections of the blockchain state - there are no external party changes or transformations here. Nodes-as-a-service providers run tons of nodes and offer them collectively through an API product, creating a stable data service.
 
-## Client diversity
+We’ve seen a ton of change in the indexing layer over the last year, with two new subcategories emerging:
 
-[Client diversity](/developers/docs/nodes-and-clients/client-diversity/) is important for the overall health of the Ethereum network because it provides resilience to bugs and exploits. There are now several client diversity dashboards including [clientdiversity.org](https://clientdiversity.org/), [rated.network](https://www.rated.network), [supermajority.info](https://supermajority.info//) and [Ethernodes](https://ethernodes.org/).
+- Forks-as-a-Service
+  - Fork any contract and add events and calculations, and then pull this data from a new “forked” RPC/data service.
+  - Some of the main providers for this are [shadow.xyz](https://www.shadow.xyz/), [ghostlogs](https://ghostlogs.xyz/), and [smlXL](https://smlxl.io/).
+  - ilemi gave his thoughts on [shortcomings and difficulties](https://twitter.com/andrewhong5297/status/1732230186966413484) on this approach.
 
-## Dune Analytics {#dune-analytics}
+- Rollups-as-a-Service (RaaS):
+  - The big theme of the year has been rollups, with Coinbase kicking it off by launching a rollup (Base) on the Optimism Stack (OP Stack) earlier this year.
```

Comment:
As you can see, time passes fast... using terms like "earlier this year" quickly become dated. Ideally we rephrase to keep content a bit more future-proof, although I do appreciate the note at the top mentioning the timeframe we're discussing.